test/ILLink.RoslynAnalyzer.Tests/TestChecker.cs

```diff
@@ -75,13 +92,10 @@ private void ValidateExpectedWarningAttribute (AttributeSyntax attribute)
 
 			Assert.True (
 				DiagnosticMessages.Any (mc => {
-					if (mc.Id != expectedWarningCode)
-						return true;
```

Comment:
Why don't we keep verifying that the warning codes match?

Reply:
Just realized that I had an idea and implemented another. Basically, I wanted to do this check along with checking that the id is supported by the current analyzer. Need to go over it again to make it work correctly.